freva-data-portal-worker/src/data_portal_worker/load_data.py

```diff
@@ -4,20 +4,16 @@
 import os
 import time
 from dataclasses import dataclass
-from typing import Any, Dict, Literal, Optional, TypedDict, cast
+from typing import Any, Dict, Literal, Optional, Tuple, TypedDict, cast
 
 import cloudpickle  # fades cloudpickle
 import redis  # fades redis
 import xarray as xr  # fades xarray
 from dask.distributed import LocalCluster  # fades distributed
 from dask.distributed import Client
 from xarray.backends.zarr import encode_zarr_variable
-from xpublish.utils.zarr import (  # fades xpublish
-    create_zmetadata,
-    encode_chunk,
-    get_data_chunk,
-    jsonify_zmetadata,
-)
+from xpublish.utils.zarr import create_zmetadata  # fades xpublish
```

Comment:
would be more clear to turn the changes back here

Reply:
That wasn't me, that was my linter.